package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.8.0",
   "dependencies": {
     "browserify": "5.9.1",
+    "browserify-optional": "^1.0.0",
```

Comment:
No worries.

I ended up making a couple of other changes to get browserid-crypto to play nicely as a client side module. I'll create a separate pull request for them so you can handle them independently or roll them up.